### PR TITLE
[CLA] update Dynapps N.V. CLA agreement

### PR DESCRIPTION
**Pull request by PieterPaulussen**
Update of DynApps N.V. corporate CLA agreement.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mart-e** [comment]
Hello,

Thank you for the CLA signature but it seems there is still an issue.

- you are signing _using_ the address `p...@me.com`
- you are signing _for_ the address `p...@dynapps.be`

The email address in the CLA must be the same as for the signature for the verification to succeed.
You can either change the your [git configuration](https://help.github.com/articles/setting-your-commit-email-address-in-git/) or modify the email in the CLA signature.

Thanks

**Comment by PieterPaulussen** [comment]
@mart-e Thanks for noticing. Updated the address..

File: doc/cla/corporate/dynapps.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ declaration.
 
 Signed,
 
-Stijn Houben stijn.houben@dynapps.be https://github.com/stijnh92
+Pieter Paulussen pieter_paulussen@me.com https://github.com/PieterPaulussen
```
Comment (mart-e) [inline doc/cla/corporate/dynapps.md]: Are you sure you don't want to do the other way around? Use your corporate email for your corporate CLA. You can manage multiple emails per github account if you wish at https://github.com/settings/emails

**Comment by mart-e** [comment]
Please let me know what you want for the emails

**Comment by PieterPaulussen** [comment]
@mart-e Nice suggestion. Updated.

**Comment by mart-e** [comment]
Well, still making the commits with the `@me.com` email address.
I have merged and changed the email to the correct one but please, make sure you change your email in your git configuration or in your [github profile](https://github.com/settings/emails)

**Pull request closed without merging**